### PR TITLE
Skip http://pie.dev tests when offline

### PR DESCRIPTION
**Pull request by hroncok**

### CODE REVIEW

**Review by BoboTiG — APPROVED** [review]
Clever, thanks!

File: tests/conftest.py [hunk 1]
```diff
@@ -22,3 +23,22 @@ def httpbin_secure_untrusted(monkeypatch, httpbin_secure):
...
+        return True
+    except OSError:
+        return False
+
+
+@pytest.fixture(scope='function')
+def httpbin_with_chunked_support(_httpbin_with_chunked_support_available):
+    if _httpbin_with_chunked_support_available:
+        return 'http://pie.dev'
+    pytest.skip('pie.dev not resolvable')
```
Comment (jakubroztocil) [inline tests/conftest.py:44]: Could we change this so that `pie.dev` is not hard-coded but rather taken from the `HTTPBIN_WITH_CHUNKED_SUPPORT` constant?

**Review by jakubroztocil — CHANGES_REQUESTED** [review]
Nice solution! Just one tweak needed.

File: tests/conftest.py (hunk 1 above)
Comment (hroncok) [reply tests/conftest.py:44]: Sure, on it.

Comment (hroncok) [reply tests/conftest.py]: Done.

**Pull request merged**